packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -45,11 +47,15 @@ exports.sourceNodes = async (
     cache,
     getCache,
     reporter,
+    schema,
   },
   pluginOptions
 ) => {
-  const { createNode, deleteNode, touchNode, setPluginStatus } = actions
-
+  const { createNode, deleteNode, touchNode, createTypes } = actions
+  const client = createClient({
```

Comment:
Since this is only used in one position, perhaps it should be created there? It looks a little awkward like this. If nothing else, you could put a comment here clarifying why the fake client was created in the first place.

Reply:
👍